lib/resources/majors.tsx

```diff
@@ -2711,6 +2711,308 @@ const bsMolCell: Major = {
+            'AS.171.104 General Physics/Biology Majors II <br />' +
+            'AS.171.108 General Physics for Physical Science Majors (AL)',
+          required_credits: 4,
+          criteria: 'AS.171.102[C]^OR^AS.171.104[C]^OR^AS.171.108[C]',
+        },
+        {
+          description:
+            '<b>Physics Laboratory I</b> <br /> AS.173.111 General Physics Laboratory I',
+          required_credits: 1,
+          criteria: 'AS.173.111[C]',
+        },
+        {
+          description:
+            '<b>Physics Laboratory II</b> <br /> AS.173.112 General Physics Laboratory II',
+          required_credits: 1,
+          criteria: 'AS.173.112[C]',
+        },
+      ],
+    },
+    {
+      name: 'Biology Sequence',
+      required_credits: 6,
+      min_credits_per_course: 3,
+      pathing: true,
+      description:
+        'Select one biology course with lab of the following. Exam credit not applicable. For more detail please visit ' +
+        'https://krieger.jhu.edu/neuroscience/bs-program/requirements/',
+      criteria:
+        'AS.020.151[C]^OR^AS.020.153[C]^OR^' +
+        'AS.020.152[C]^OR^AS.020.154[C]^OR^' +
+        'AS.020.305[C]^OR^AS.020.315[C]^OR^AS.250.253[C]^OR^AS.250.254[C]^OR^' +
+        'AS.020.303[C]^OR^AS.020.340[C]^OR^' +
+        'AS.020.306[C]^OR^AS.020.316[C]^OR^' +
+        'AS.020.374[C]^OR^AS.020.377[C]',
+      fine_requirements: [
+        {
+          required_credits: 6,
+          description: '<b>Gen Bio I with lab</b>',
+          criteria: 'AS.020.151[C]^AND^AS.020.153[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Gen Bio II with lab</b>',
+          criteria: 'AS.020.152[C]^AND^AS.020.154[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Biochem with lab</b>',
+          criteria:
+            '(AS.020.305[C]^AND^AS.020.315[C])^OR^AS.250.253[C]^OR^AS.250.254[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Genetics with lab</b>',
+          criteria: 'AS.020.303[C]^AND^AS.020.340[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Cell Bio with lab</b>',
+          criteria: 'AS.020.306[C]^AND^AS.020.316[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Comparative Physiology with lab</b>',
+          criteria: 'AS.020.374[C]^AND^AS.020.377[C]',
+        },
+      ],
+    },
+    {
+      name: 'Focus Area',
+      required_credits: 9,
+      min_credits_per_course: 1,
+      pathing: true,
+      description:
+        'Nine credits of 300-level or higher approved courses from one of four focus areas: systems neuroscience, ' +
+        'cognitive neuroscience, computational neuroscience, or cellular and molecular neuroscience',
+      criteria:
+        'NEUR-ST[T]^OR^NEUR-CG[T]^OR^NEUR-CP[T]^OR^NEUR-CM[T]^OR^AS.080.3[C]^OR^AS.200.3[C]',
```

Comment:
I think using `AS.080.3[C]^OR^AS.200.3[C]` does not cover all the upper level classes. Thus, classes other than 300 level will not satisfy this requirement. For example, 400 level classes are also considered upper level but will not work in this class. It is also not best practice to use parts of the class number in the criteria string. Sorry I should have mentioned that in the majors documentation. Try replacing the criteria string with something like:

`criteria: '(NEUR-ST[T]^OR^NEUR-CG[T]^OR^NEUR-CP[T]^OR^NEUR-CM[T])^AND^(Upper Level[L])',`